### PR TITLE
Moved to coroutines-1.5.1-mm-dev1 for experimental memory model

### PR DESCRIPTION
**Pull request by alexander-gorshenev**
## Proposed Changes

  -
  -
  -

## Testing

Test: Describe how you tested your changes. Note that this line (with `Test:`) is required, your PR will not build without it!

## Issues Fixed

Fixes: [Optional] The bug on [https://issuetracker.google.com](https://issuetracker.google.com) being fixed

### CODE REVIEW

File: compose/ui/ui-graphics/build.gradle [hunk 1]
```diff
@@ -65,11 +65,35 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
...
+                "-linker-option", "-framework", "-linker-option", "Metal",
+                "-linker-option", "-framework", "-linker-option", "CoreText",
+                "-linker-option", "-framework", "-linker-option", "CoreGraphics",
+                "-linker-option", "-framework", "-linker-option", "CoreServices"
+            ]
+            configure([macosX64, macosArm64]) {
+                binaries.all {
+                    freeCompilerArgs += darwinFlags
+                    // TODO: the current release binary surprises LLVM, so disable checks for now.
+                    freeCompilerArgs += "-Xdisable-phases=VerifyBitcode"
```
Comment (olonho) [inline compose/ui/ui-graphics/build.gradle]: Why this change is needed?

**Review by olonho — APPROVED** [review]

File: compose/ui/ui-graphics/build.gradle (hunk 1 above)
Comment (alexander-gorshenev) [reply compose/ui/ui-graphics/build.gradle]: For the test binary. I have cleaned it up a little.

**Review by olonho — APPROVED** [review]
